### PR TITLE
use webgl in plotly.py

### PR DESCRIPTION
**Pull request by Saksham20**
using webgl for plotting multitrace plots

### CODE REVIEW

**Comment by Saksham20** [comment]
here is the working gif for timeseries and spatialseries:
plotlygl

File: nwbwidgets/controllers/group_and_sort_controllers.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def order_dd_observer(self, change):
         if change["name"] == "value":
             self.order_by = self.order_dd.value
 
-            order_vals = self.get_column_values(by=self.order_by)
+            order_vals = self.get_column_values(column_name=self.order_by)
```
Comment (Saksham20) [inline nwbwidgets/controllers/group_and_sort_controllers.py:257]: this was a small unrelated fix that got into this PR somehow.  I didn't bother making another PR just for this.

**Review by bendichter — APPROVED** [review]

**Review by bendichter — APPROVED** [review]
